androguard/core/bytecodes/apk.py

```
@@ -770,6 +830,8 @@ def get_android_resources(self):
                 self.arsc["resources.arsc"] = ARSCParser(self.zip.read("resources.arsc"))
                 return self.arsc["resources.arsc"]
             except KeyError:
+                import traceback
```

why do you add that ? Let's raise a new exception. Add a new class for that.